p2p/host/resource-manager/conn_limiter.go

```diff
@@ -74,11 +74,16 @@ func sortNetworkPrefixes(limits []NetworkPrefixLimit) []NetworkPrefixLimit {
 // for a specific subnet than the default limit per subnet.
 func WithNetworkPrefixLimit(ipv4 []NetworkPrefixLimit, ipv6 []NetworkPrefixLimit) Option {
 	return func(rm *resourceManager) error {
+		connLimiterInstance, ok := rm.connLimiter.(*connLimiter)
+		if !ok {
+			return nil // early exit, it might be a custom implementation
```

Comment:
or maybe throw an error here? Just to signal that the method call did not produce the expected result. Valid also on L100

Reply:
removed